SYCL/SubGroupMask/GroupSize.cpp

```diff
@@ -82,9 +82,30 @@ int main() {
 #ifdef SYCL_EXT_ONEAPI_SUB_GROUP_MASK
   queue Queue;
 
-  test<8>(Queue);
-  test<16>(Queue);
-  test<32>(Queue);
+  // Check supported subgroup size before calling to make sure platform supports it
+  for (const auto& x :
+     Queue.get_device().get_info<info::device::sub_group_sizes>()) {
+
+    switch(x){
+    case 4:
+	test<4>(Queue);
+	break;
+    case 8:
+        test<8>(Queue);
+        break;
+    case 16:
+        test<16>(Queue);
+        break;
+    case 32:
+        test<32>(Queue);
+	break;
+    case 64:
+	test<64>(Queue);
```

Comment:
```suggestion
	test<64>(Queue);
	break;
```
Otherwise we will have it fall through to the default case and print that it isn't supported. Not the end of the world, but potentially confusing.

Reply:
@steffenlarsen .... Thank you for catching that...
Resolved